private/buf/cmd/buf/command/beta/convert/convert.go

```diff
@@ -142,7 +134,11 @@ func run(
 	if err != nil {
 		return err
 	}
-	payloadMessageRef, err := bufconvert.NewMessageEncodingRef(ctx, flags.Payload, bufconvert.MessageEncodingBin)
+	payload, err := bufcli.GetInputValue(container, flags.InputHashtag, "-")
```

Comment:
whats "-" about?

Reply:
"-" indicates that it's being read from stdin instead of the arg or flag